apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -31,8 +31,6 @@ export const PropertyName = ({ property, label, css }: PropertyProps) => {
                   color: "$hiContrast",
                 }),
           }}
-          variant="contrast"
-          size="1"
```

Comment:
have you checked if this looks correct according to figma after the change?

Reply:
Didn't check figma, but it didn't change compared to `main`